src/modules/Host.rb

```diff
@@ -121,27 +110,20 @@ def EnsureHostnameResolvable
     # Read hosts settings
     # @return true if success
     def Read
-      return true if @initialized == true
+      return true if @initialized
+
 
       # read /etc/hosts
-      if Ops.greater_than(SCR.Read(path(".target.size"), @hosts_file), 0)
-        hostlist = SCR.Dir(path(".etc.hosts"))
-        @hosts = Builtins.listmap(hostlist) do |host|
-          names = Convert.convert(
-            SCR.Read(
-              Builtins.topath(Builtins.sformat(".etc.hosts.\"%1\"", host))
-            ),
-            from: "any",
-            to:   "list <string>"
-          )
-          next { host => names } if names != []
-        end
+      if Ops.greater_than(SCR.Read(path(".target.size"), CFA::Hosts::PATH), 0)
```

Comment:
well, cfa currently do not have method to recognize if file is not empty or exists. It raise exception. Of course I can use some cli tool to get it